packages/interactive_media_ads/pigeons/interactive_media_ads_ios.dart

```diff
@@ -291,6 +291,20 @@ enum FriendlyObstructionPurpose {
   unknown,
 }
 
+/// Different UI elements that can be customized.
+///
+/// See https://developers.google.com/ad-manager/dynamic-ad-insertion/sdk/ios/reference/Enums/IMAUiElementType.html.
+enum UiElementType {
```

Comment:
Nit: UIElementType. https://dart.dev/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words